examples/routing_service/routing_service_constrained_wan/README.md

```diff
@@ -0,0 +1,3 @@
+These files are used to configure a constrained WAN demo that is available on RTI's community 'examples' website
```

Comment:
Please, could you add a link to the example?

Reply:
The example on the RTI community site does not exist yet, because hosting the xml files on github is a prerequisite. You can see what the demonstration will cover by looking at the existing internal confluence link: /display/FAE/Routing+Service+Constrained+Wan+Demo

I will update the readme.md when the example exists, and include the link

Reply:
I've created an issue to update the readme file once it is in GitHub. Anyway, I don't think that make so much sense because if somebody gets the example from GitHub, they already know the link to the example. I think we should example a little bit what the example does (the scenario that is run, etc...)